### ISSUE TITLE
Maintenance of mopidy-soudcloud and other unmaintained plugin

### ISSUE THREAD

**Issue by ticosax** [mopidy/mopidy-soundcloud#85]
Hi @jodal,
I'd like to bring your attention to the [jazzband organization](https://jazzband.co/) where several users could collaborate. It is important to raise that plenty of good pull request are waiting to be merged, ignoring them is ignoring as many contributors. This project is going to die if we don't allow more people to be involved.
I could make the pull request removing the section ``Maintainer Wanted`` but I don't believe it is going to work because the story will repeat itself, one maintainer is simply not enough.
There plenty of testimonies over the web that tell the same story. Granting write access to everyone makes the project better. Jazzband should suffice to convince you.

What are your thoughts ?

PS:
Some inspirational links:
http://felixge.de/2013/03/11/the-pull-request-hack.html
http://jakewins.com/p/clickbait
https://rfc.zeromq.org/spec:22/
http://hintjens.com/blog:106

**Comment by amki** [comment]
It's been nearly 2 months for this and 2,5 since the last commit was done @jodal .

What about mopidy's soundcloud plugin?

**Comment by gutierri** [comment]
@ticosax Fork to Jazz Band and we keep in parallel.

**Comment by ticosax** [comment]
@gutierri The goal is to encourage collaboration and share the burden of maintaining OSS projects.
We will need to be able releases to pypi or make sure packages for distros are picking up code from upstream repository as well, something that a fork can't do. We really need commitment from maintainers of mopidy for this to work IMO.

**Comment by jodal** [comment]
> *This was generated by AI during triage.*

Mopidy-SoundCloud stays in the mopidy organization, where it gets maintenance updates. A move to Jazzband is not planned. If you want to help maintain it, please say so in a new issue or on the forum. Closing.
